### DISCUSSION TITLE
NullPointerException in startPlayer

### DISCUSSION THREAD

**Discussion by mojtabaasadi** [dooboolab/react-native-audio-recorder-player#481] [Q&A]
Hi , thanks for your efforts on this project,
I am wondering why it doesn't reject with  error when `NullPointerException` happens?

https://github.com/hyochan/react-native-audio-recorder-player/blob/613e8fa333e738d91ebbc092ae6ab1ceb364f774/android/src/main/java/com/dooboolab.audiorecorderplayer/RNAudioRecorderPlayerModule.kt#L263

**Comment by hyochan** [comment]
@mojtabaasadi Thanks for the great question.

I've lately, thrown the `error` in the previous version. However, many RN devs were annoyed by the crashes which happened very often because when the player stops, the `mediaPlayer` will be null.

Therefore, I've chosen to give the `log` instead.

Please share any further opinions
